.github/pull_request_template.md

```diff
@@ -24,6 +24,7 @@ Fixes: < Link to related issues that will be fixed by this pull request, if they
  - [ ] Document any changes to public APIs.
  - [ ] Apply changelog label ('breaking change', 'bug :beetle:', 'build', 'docs', 'feature :green_apple:', 'performance :zap:', 'testing :100:') or use the label 'skip changelog'
  - [ ] Add an entry inside this element for inclusion in the `mapbox-maps-android` changelog: `<changelog></changelog>`.
+ - [ ] Port this PR to `v10.[version]` release branch if this change should be included in upcoming `v10.[version]-rc.[number]` or `v10.[version]` stable release.
```

Comment:
It's only about time span when release have alread been started but not yet finished, right?

Reply:
How to `port` here? Cherry-pick?

Reply:
what if it's a PR towards the release branch, shall we mention to port the fix to the main branch as well?(e.g change log for rc and final releases.)

Reply:
@pengdev - imho it's impossible situation, if we have some branch targeted to release branch, we firstly prepare PR to `main` in any case and then port it to release branch

@Chaoba - I did not mention cherry-pick as situations may happen when cherry-pick will be impossible due to some conflicts between `main` branch and release branch. It's PR author's responsibility to select correct way here, in 90% of cases cherry-pick should be enough but sometimes PR should be open explicitly.

@yunikkk no, it's about 4-week release adoption time. When we cut `beta` for some release - this point I've added becomes valid till we release stable version. Our regular schedule is `beta` -> `rc` after 2 weeks -> stable version after 2 weeks more.

Reply:
```suggestion
 - [ ] Fixes for the release branch should be merged to `main` branch first and then ported to the release branch.
   - [ ] i.e. the PR should be ported to `v10.[version]` release branch if this change should be included in upcoming `v10.[version]-rc.[number]` or `v10.[version]` stable release.
```